opensciencegrid/vo-frontend/Dockerfile

```diff
@@ -24,6 +24,11 @@ RUN yum -y install \
       vo-client && \
     yum -y clean all
 
+# for developement, grab gwms from upcoming
+RUN if [[ $BASE_YUM_REPO = development ]]; then \
+      yum -y --enablerepo=osg-upcoming-testing upgrade glideinwms-vofrontend-standalone; \
+    fi
+
```

Comment:
I would move this into the previous RUN block, before the `yum -y clean all`, so we don't populate the yum cache twice (and leave it around afterward).

Reply:
Ok, check now please